outputs/costs_2020.csv

```diff
@@ -25,17 +90,55 @@ HVDC inverter pair,lifetime,40.0,years,Hagspiel, from old pypsa cost assumptions
 HVDC overhead,FOM,2.0,%/year,Hagspiel, from old pypsa cost assumptions
 HVDC overhead,investment,400.0,EUR/MW/km,Hagspiel, from old pypsa cost assumptions
 HVDC overhead,lifetime,40.0,years,Hagspiel, from old pypsa cost assumptions
-HVDC submarine,FOM,2.0,%/year,Hagspiel, from old pypsa cost assumptions
-HVDC submarine,investment,2000.0,EUR/MW/km,Own analysis of European submarine HVDC projects since 2000, from old pypsa cost assumptions
-HVDC submarine,lifetime,40.0,years,Hagspiel, from old pypsa cost assumptions
-OCGT,FOM,1.78,%/year,"Danish Energy Agency, technology_data_for_el_and_dh_-_0009.xlsx",52 OCGT - Natural gas:  Fixed O&M
+HVDC submarine,FOM,0.35,%/year,Purvins et al. (2018): https://doi.org/10.1016/j.jclepro.2018.03.095 .,
+HVDC submarine,investment,471.161167,EUR/MW/km,Purvins et al. (2018): https://doi.org/10.1016/j.jclepro.2018.03.095 .,
+HVDC submarine,lifetime,40.0,years,Purvins et al. (2018): https://doi.org/10.1016/j.jclepro.2018.03.095 .,
```

Comment:
Any background info on this change?

Reply:
During a discussion someone pointed out to me that submarine HVDC cables have gotten a lot cheaper and can at today's cost already be similar to overhead HVDC. This is an updated cost assumption with newer reference.